packages/jest-worker/src/Farm.ts

```diff
@@ -56,12 +56,12 @@ export default class Farm {
       customMessageListeners.push(listener);
       return () => {
         customMessageListeners = customMessageListeners.filter(
-          l => l !== listener,
+          oncustomMessage => oncustomMessage !== listener,
```

Comment:
```suggestion
          onCustomMessage => onCustomMessage !== listener,
```

If you want to rename this, let's be consistent with the capitalisation used elsewhere, otherwise this could lead to subtle bugs (and confusing editor autocompletions)

Reply:
`customListener` to match name of array more closely?

Reply:
Yeah, my bad missed the capitalization there, but `customListener` sounds cool